client/src/utils.js

```diff
@@ -0,0 +1,61 @@
+export const formatDateforAPI = (currDate) => {
+  // getMonth returns 0-11
+  const currMonthStr = String(currDate.getMonth() + 1).padStart(2, "0");
+  // getDate returns 1-31
+  const currDayStr = String(currDate.getDate()).padStart(2, "0");
+  return currDate.getFullYear().toString() + currMonthStr + currDayStr;
+};
+
+export const buildDateRangeRoute = (startDate, endDate) => {
+  return `/api/tasks?start=${formatDateforAPI(
+    startDate
+  )}&end=${formatDateforAPI(endDate)}`;
+};
+
+const getDateTime = (taskDateISOString) => {
```

Comment:
For the getDateTime function, I think we can move this into the TaskCard component since this a rather specific format for this component.

Reply:
Ideally we should have any date related formatting stuff within the component and simply pass in whatever the API returns to us.

Reply:
My initial design was to let TaskCard just handle displaying certain strings. As such, all the props are designed this way. Taking your comment into consideration, I've moved all the processing into `TaskListView` instead. So, the pages will only store the API result, and `TaskListView` will receive the API result.

Reply:
Right! Normally we would have the TaskCard or any other component just display the data that we have but since this date format is specific for the component, it would be more ideal to have it within the component!